app/src/main/java/com/concordia/mcga/bottomsheet/BottomSheet.java

```diff
@@ -245,12 +245,12 @@ public boolean onLayoutChild(CoordinatorLayout parent, V child, int layoutDirect
         mParentHeight = parent.getHeight();
 
         if (mType == "building_navigation") {
-            mMinOffset = mParentHeight - child.getHeight() / 2 - child.getHeight() / 5 - child.getHeight() / 10;
-            mMaxOffset = mParentHeight - parent.getHeight() / 6;
+            mMinOffset = mParentHeight - child.getHeight() / 2 - child.getHeight()/5 - child.getHeight() / 40;
+            mMaxOffset = mParentHeight - parent.getHeight() / 7;
```

Comment:
Im running it on my emulator and the bottom sheet is  flush. So I'm not sure about these value working for all screens sizes